ipam/ippool_manager.go

```diff
@@ -518,3 +509,10 @@ func addOffsetToIP(ip, endIP net.IP, offset int) (net.IP, error) {
 	copy(ip[16-IPBytesLen:], IPBytes)
 	return ip, nil
 }
+
+// formatAddressName renders the name of the IPAddress objects
+func (m *IPPoolManager) formatAddressName(address ipamv1.IPAddressStr) string {
```

Comment:
It looks okay for me, although I am not that familiar with base code in this repo, but `address` shouldn't be called here as `addresses`

Reply:
it's only one at a time

Reply:
Makes sense